app/src/ActionModal/components/ActionModal.vue

```diff
@@ -699,6 +699,9 @@ export default {
           network: this.network,
           signingType: this.selectedSignMethod,
           password: this.password,
+          polkadotAPI: this.$store.state.connection.polkadotAPIs[
+            this.network.id
+          ],
```

Comment:
Here you can also call `getPolkadotApi`

Reply:
but it is already in store? Then where is the point?

Reply:
don't put it into the store. why maintain it twice?